README.md

```diff
@@ -1,2 +1,35 @@
 # hazzat-api
-Hazzat backend REST API
+![logo](http://www.hazzat.com/portals/_default/skins/hazzat/images/logo.png "Hazzat Logo")
+
+Hazzat backend REST API writen in vanilla Node.js with a React frontend. [www.hazzat.com](http://hazzat.com)
+
+## Prerequisites
+
+* Node.js & npm
+* SQL Server
+
+## Build
+
+> * `npm install`
+> * `npm run build`
+
+## Run
+
+> `node app`
+
+
+## Contributing
+
+Anyone can contribute to this open source project! Just open a Github issue and fork a fresh copy.
+
+## License 
+
+All work in this repo is covered under the MIT License. You should have received a copy of the Licence with the work therin. If not, you can obtain a license from [opensource.org](https://opensource.org/licenses/MIT).
+
+
+## Community
+
+* [Slack](https://join.slack.com/t/hazzat/shared_invite/enQtNjU0MTAxMDc5ODYzLTUxOWNhOGE2OGExNzAxYjQwNDBhZWMwOTExMjRhMDNkN2Q2ZGY1MDRhMjc1ZDhhNmFiYjA2NDYxZjJkMTFlN2M)
```

Comment:
Is it a good idea to leave this available online? Would this mean anybody can join anonymously and start watching the discussion? Not sure if this is normal or not, apologies if this is standard practice

Reply:
Anyone can view the channel discussions online, only peer to peer discussions are kept private. Typically this is done on some larger open source projects.